obspy/clients/fdsn/__init__.py

```diff
@@ -134,8 +134,69 @@
                                         endtime=endtime)
         inventory.plot()
 
+
+Basic FDSN FedCatalog Client Usage
+----------------------------------
+
+The
+:mod:`FDSN fedcatalog_client <obspy.clients.fdsn.routers.fedcatalog_client>`
```

Comment:
This link seems [broken in the docs build](http://docs.obspy.org/pull-requests/1779/packages/obspy.clients.fdsn.html#basic-fdsn-fedcatalog-client-usage), I think the reason is that the new submodules are not added to the sphinx package skeleton: https://github.com/obspy/obspy/blob/master/misc/docs/source/packages/obspy.clients.fdsn.rst

See also the build log for respective warnings/errors: http://docs.obspy.org/pull-requests/1779/log.txt